crypto/bio/bss_sock.c

```diff
@@ -307,55 +307,24 @@ int BIO_sock_should_retry(int i)
     return 0;
 }
 
+
 int BIO_sock_non_fatal_error(int err)
 {
-    switch (err) {
 # if defined(OPENSSL_SYS_WINDOWS)
-#  if defined(WSAEWOULDBLOCK)
-    case WSAEWOULDBLOCK:
-#  endif
-# endif
-
-# ifdef EWOULDBLOCK
-#  ifdef WSAEWOULDBLOCK
-#   if WSAEWOULDBLOCK != EWOULDBLOCK
-    case EWOULDBLOCK:
-#   endif
-#  else
-    case EWOULDBLOCK:
-#  endif
-# endif
-
-# if defined(ENOTCONN)
-    case ENOTCONN:
+    return (err == WSAEWOULDBLOCK ||
+	err == WSAENOTCONN ||
+	err == WSAEINTR ||
+	err == WSAEINPROGRESS ||
+	err == WSAEALREADY);
+# else
+    return (err == EWOULDBLOCK ||
+	err == EAGAIN ||
+ 	err == ENOTCONN ||
+	err == EINTR ||
+	err == EPROTO ||
+	err == EINPROGRESS ||
+	err == EALREADY);
```

Comment:
our coding style says the `||` operator should be on the beginning of the next line and indented. Also we do not put parentheses around `return` value.